### PR TITLE
Add aarch64 support

### PR DESCRIPTION
**Pull request by statickidz**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-spotube/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

This PR adds support to build the ARM image and support those architecture.

## Benefits of this PR and context:

Could run on ARM.

## How Has This Been Tested?

Building the image locally.

### CODE REVIEW

File: README.md
Comment (thespad) [inline README.md:1]: This file is templated, changes need to be made to the readme-vars.yml

File: Jenkinsfile
Comment (thespad) [inline Jenkinsfile:1]: This file is templated, changes need to be made to the jenkins-vars.yml

File: Dockerfile.aarch64 [hunk 1]
```diff
@@ -0,0 +1,43 @@
+FROM ghcr.io/linuxserver/baseimage-kasmvnc:arm64v8-kali
```
Comment (thespad) [inline Dockerfile.aarch64]: The base image needs to match the amd64 release.

**Review by thespad — CHANGES_REQUESTED** [review]

File: README.md
Comment (statickidz) [reply README.md:1]: done

File: Jenkinsfile
Comment (statickidz) [reply Jenkinsfile:1]: done

File: Dockerfile.aarch64 (hunk 1 above)
Comment (statickidz) [reply Dockerfile.aarch64]: Changed to the `baseimage-kasmvnc:arm64v8-debianbookworm` to match the one used in the main Dockerfile but in arm64, I hope is ok like this

**Comment by thespad** [comment]
All looks good, please add an entry to the readme changelog reflecting the change and then I'm happy to merge.

Because the CI won't run a Jenkinsfile from a forked repo I'm going to do a local build and test myself on arm64 just to make sure it all looks OK.

**Comment by statickidz** [comment]
> All looks good, please add an entry to the readme changelog reflecting the change and then I'm happy to merge.

It should be done now!

> Because the CI won't run a Jenkinsfile from a forked repo I'm going to do a local build and test myself on arm64 just to make sure it all looks OK.

Sure, I've noticed that on the Jenkins pipeline.

File: readme-vars.yml [hunk 2]
```diff
@@ -87,4 +87,5 @@ app_setup_block: |
 
 # changelog
 changelogs:
+  - { date: "10.09.24:", desc: "Add aarch64 support." }
```
Comment (thespad) [inline readme-vars.yml]: Date is the wrong way around, DD.MM.YY :)

Comment (statickidz) [reply readme-vars.yml]: done, used to the US date format :D

Comment (thespad) [reply readme-vars.yml]: Whatever change you made has weirdly reformatted the readme-vars

Comment (statickidz) [reply readme-vars.yml]: Sorry for that, format on save feature wanted to do that, I've to turn if off for that particular file, should be fine now.e

**Review by thespad — APPROVED** [review]

**Pull request merged**